src/mainloop.rkt

```diff
@@ -413,24 +519,45 @@
 (define (extract!)
   (define repr (*output-repr*))
   (define all-alts (atab-all-alts (^table^)))
-  (*all-alts* all-alts)
-  (define joined-alt
+
+  ; Constant alts
+  (define const-alts
+    (let ([prog (alt-program (car all-alts))])
+      (list (make-alt `(λ ,(program-variables prog) 1))
+            (make-alt `(λ ,(program-variables prog) 0))
+            (make-alt `(λ ,(program-variables prog) -1)))))
```

Comment:
... do these help? How often? Which ones?

Reply:
In single output (normal) mode, there are very few where this matters (see Rump's example in numerics - the expression directly evaluates to 1). We've discussed "average output" programs before, or perhaps "limit at infinity" programs. There might be a better way to do this then directly injecting them. 

For Pherbie, this helps generate a lot of low cost alts. For programs that converge to 0 or 1 for large (or small) inputs, these often get chosen. I'll add this to the Trello, so we remember.